### DISCUSSION TITLE
filling a variable from within cy.wait()

### DISCUSSION THREAD

**Discussion by AbrahamBrookes** [cypress-io/cypress#19497] [Questions and Help]
what am I misunderstanding here? 
![image](https://user-images.githubusercontent.com/8340776/147719531-6e16f1ae-cca1-4444-a811-3f719dbec6c8.png)

**Comment by bahmutov** [comment]
@AbrahamBrookes https://www.youtube.com/watch?v=uGWWoq0ECY8
